example-apps/trigger/README.md

```diff
@@ -2,6 +2,4 @@
 
 [![Build Status](https://travis-ci.org/zapier/zapier-platform-example-app-trigger.svg?branch=master)](https://travis-ci.org/zapier/zapier-platform-example-app-trigger)
 
-A barebones app that has a resource defined.
-
-> We recommend using the zapier-platform-cli and `zapier init . --template=trigger` to create an app.
```

Comment:
This is no longer supported.

Reply:
This is replaced by `zapier init .` and `zapier scaffold trigger my_trigger`